modules/backend/widgets/Lists.php

```diff
@@ -1544,6 +1544,13 @@ protected function getSortColumn()
         return $this->sortColumn;
     }
 
+    /*
+     * Returns the current sort direction or default of 'asc'
+     */
+    public function getSortDirection () {
```

Comment:
@empower-josh Please place the brackets next to the method name, and put the curly brace on a new line as PSR-2 guidelines.
```suggestion
    public function getSortDirection()
    {
```